generators/app/index.js

```diff
@@ -77,7 +77,7 @@ module.exports = yeoman.Base.extend({
 
     // Testing folder / stub
     if (props.addTests) {
-      var testPath = folder + '/__test__/index.js';
+      var testPath = folder + '/__test__/' + data.name + 'Spec.js';
```

Comment:
This is up for debate but what I personally prefer `xxxxx.test.js` vs `xxxxxSpec.js` as I feel it's a bit more clear. I think this is one of those instances where we get team feedback on naming conventions. Won't stop me from merging this in, in the meantime.